.gitignore

```diff
@@ -109,4 +109,22 @@ venv.bak/
 
 # own files
 start.bat
-config-local.ini
\ No newline at end of file
+config-local.ini
+
+
+
+.idea/ALAuto.iml
+.idea/inspectionProfiles/profiles_settings.xml
+.idea/misc.xml
+.idea/modules.xml
+.idea/rSettings.xml
+.idea/vcs.xml
+
+
+config_dev.ini
+screenshot.png
+screenshot1.png
+screenshot2.png
+screenshot3.png
+screenshot4.png
+screenshot5.png
```

Comment:
You could just add the whole .idea folder rather than every single file on it by adding `.idea/`. Also, why use config_dev.ini when there's already a config-local.ini added, just rename your existing one. 
Finally, I think all those screenshots could be moved into another folder, whether inside the project folder or not, to avoid adding each single one of them.

Reply:
i'll fix, sorry.